docs/user/install-gke.md

```diff
@@ -81,4 +81,99 @@ gcloud container clusters create CLUSTER_NAME \
+```
+PROMETHEUS_ENABLED = True
+PROMETHEUS_ADDR = '0.0.0.0'
+PROMETHEUS_PORT = 9200
+```
+Please ensure `PROMETHEUS_ENABLED` is set to `True` and that the `PROMETHEUS_PORT` matches the `prometheus.io/port` section in the `turbinia-worker.yaml` and `turbinia-server.yaml` as well as matching ports in the `turbinia-server-metrics-service.yaml` and `turbinia-worker-metrics-service.yaml` GKE deployment files.
+
+### Deploying Prometheus
+In this deployment method, we are using [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) to deploy the Prometheus stack to the cluster. This simplifies the setup required and automatically deploys Prometheus, Grafana, and Alert Manager to the cluster through manifest files. Before proceeding with the setup, please ensure you are connected to the cluster with Turbinia deployed and can run commands via `kubectl`, then proceed to the following steps to configure Prometheus with Turbinia.
+
+* Clone the github repo [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) locally. Please ensure that the branch  cloned is compatible with your Kubernetes cluster version else you may run into issues. Please see the [Compatibility Matrix](https://github.com/prometheus-operator/kube-prometheus) section of the repo for more details.
+* Once cloned, run the following commands to deploy the stack
+    * `kubectl create -f manifests/setup`
+    * `kubectl create -f manifests/`
+* Create a secret from the file `prometheus-additional.yaml` located in the Turbinia folder.
+    * `kubectl create secret generic additional-scrape-configs --from-file=monitoring/k8s/prometheus/prometheus-additional.yaml --dry-run=client -oyaml > additional-scrape-configs.yaml`
+* You will then need to update the `prometheus-prometheus.yaml` file located in the `kube-prometheus/manifests` folder with this extra scrape config
+    ```
+    additionalScrapeConfigs:
+        name: additional-scrape-configs
+        key: prometheus-additional.yaml
+    ```
+* Once complete apply the changes made through the following commands
+    * `kubectl -n monitoring apply -f additional-scrape-configs.yaml`
+    * `kubectl -n monitoring apply -f manifests/prometheus-prometheus.yaml`
+* To get Turbinia custom rules imported, apply the `turbinia-custom-rules.yaml` file located in the Turbinia directory
+    * `kubectl -n monitoring apply -f monitoring/k8s/prometheus/turbinia-custom-rules.yaml`
+* You can test that the changes were properly made by connecting to the Prometheus console and searching for `turbinia`. If related metrics pop up in the search bar, then Turbinia metrics are properly being ingested by Prometheus. You can also check to see if the Turbinia custom rules have been applied by navigating to Status -> Rules then searching for one of the custom rule names. To connect to the Prometheus console, run the following command
+    * `kubectl -n monitoring port-forward svc/prometheus-k8s 9090`
+
```

Comment:
Maybe make this it's own step that comes before the "test that the changes were properly made..." section?

Reply:
Sounds good, added a section named **Testing Prometheus Deployment** before Grafana setup to seperate it out more.